tests/crypto/ec_tests.c

```diff
@@ -878,7 +876,17 @@ static void _test_crl_distribution_points(void)
     OE_TEST(r == OE_BUFFER_TOO_SMALL);
 
     {
-        OE_ALIGNED(8) uint8_t buffer[buffer_size];
+        uint8_t* buffer = (uint8_t*)oe_memalign(8, buffer_size);
```

Comment:
Can we not use malloc here sicne malloc already guarantee alignment to sizeof(void*)?

Reply:
Yes, you could. I'm effectively repurposing part of this test to validate that oe_memalign works correctly in host & enclave contexts is all.